Lib9c/Action/HackAndSlash.cs

```diff
@@ -15,6 +15,10 @@
 
 namespace Nekoyume.Action
 {
+    /// <summary>
+    /// Hard forked at https://github.com/planetarium/lib9c/pull/884
```

Comment:
It was recently updated. ([link](https://github.com/planetarium/lib9c/pull/921))